lib/dustoff_web/controllers/user_session_html/new.html.heex

```diff
@@ -0,0 +1,71 @@
+<Layouts.app flash={@flash} current_scope={@current_scope}>
+  <div class="mx-auto max-w-sm space-y-4">
+    <.header class="text-center">
+      <p>Log in</p>
+      <:subtitle>
+        <%= if @current_scope do %>
+          You need to reauthenticate to perform sensitive actions on your account.
+        <% else %>
```

Comment:
Feels like this `if` is making assumptions about the page being rendered with an active session. 🤷‍♂️